src/instructlab/config.py

```diff
@@ -137,6 +147,7 @@ class Config(BaseModel):
 
     # additional fields with defaults
     general: _general = _general()
+    confluence: Optional[_confluence]
```

Comment:
You also have to assign a default value to make the field an optional argument.
```suggestion
    confluence: Optional[_confluence] = None
```